js/cfapi.js

```diff
@@ -3,26 +3,25 @@
 * See https://github.com/codeforamerica/cfapi for an overview
 * See http://codeforamerica.org/api/ for full documentation
 */
-
 (function($, window) {
   $.getJSON( "http://codeforamerica.org/api/organizations/Code-for-Denver", function( data ) {
     var events = [];
 
-    $(data.current_events).each( function() {
-      var mEvent = this, name, time, image, link, linkText, tile;
-      name = "<div class='event-name'>" + mEvent.name + "</div>";
-      time = "<time datetime="+mEvent.start_time+">" + (function(){ return new Date(mEvent.start_time).toLocaleString();})() + "</time>";
-      image = "<img src='img/meetup-icon.png'/>";
-      linkText = "<p>view on meetup.com</p>";
-      link = "<a href=" + mEvent.event_url + " >" + image + linkText + "</a>";
-      tile = name + time + link;
-      events.push("<li class='event'>" + tile + "</li>");
+    //  usage: {{dateFormat myDateString}}
+    Handlebars.registerHelper('dateFormat', function(context, block) {
+      return new Date(context).toLocaleString();
     });
 
-    $( "<ul/>", {
-      "class": "my-new-list",
-      html: events.join("")
-    }).appendTo("div#meetups-container");
+    var meetupTemplate = "<ul>{{#current_events}}<li class='event'>"+
+      "<div class='event-name'>{{name}}</div>"+
+      "<time datetime={{start_time}}>{{dateFormat start_time}}</time>" +
+      "<a href='{{event_url}}'><img src='img/meetup-icon.png'/><p>view on meetup.com</p></a>" +
+      "</li>{{/current_events}}</ul>";
```

Comment:
No unauthorized white space, line breaks or anything....  has to be string concatenation  :hankey:     I wish GH had a poop emoji with a frowny face.  :hankey: + :frowning: